README.md

```diff
@@ -106,7 +74,15 @@ private static void showWalletInstallDialog(Context context, String message) {
         .show();
 }
 ```
+If you follow with the use of the code above the resources below are required.
+```
+  <string name="install">INSTALL</string>
+  <string name="skip">SKIP</string>
+  <string name="install_wallet_from_iab">To pay for in-app items, you need to install the AppCoins BDS Wallet.</string>
+  <string name="wallet_missing">AppCoins Wallet Missing</string>
+```
 
+Finally, the following method is the one used to redirect the used to AppCoins BDS Wallet store page. 
```

Comment:
Should it be "Finally, the following method is the one used to redirect the **user** to AppCoins BDS Wallet store page."?